engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -53,6 +53,11 @@ paths:
           type: file
           in: formData
         - $ref: '#/parameters/workflowTypeParam'
+        - name: workflowRoot
+          description: The root object to be run. Only necessary for CWL submissions containing multiple objects (in an array).
+          required: false
+          type: string
+          in: formData
```

Comment:
This is what I was talking about at stand up and what I mentioned yesterday @geoffjentry